soccer/gameplay/plays/offense/adaptive_formation.py

```diff
@@ -13,10 +13,19 @@
 import tactics.advance_zone_midfielder
 import skills.move
 import skills.capture
+import situational_play_selection
 
 
 class AdaptiveFormation(standard_play.StandardPlay):
 
+
+    _situationList = [
+        situational_play_selection.SituationalPlaySelector.Situation.
+        ATTACK_GOAL, situational_play_selection.SituationalPlaySelector.
+        Situation.DEFEND_CLEAR, situational_play_selection.
+        SituationalPlaySelector.Situation.OFFENSIVE_SCRAMBLE
+    ]
```

Comment:
So uhh, this is how make pretty wants to do it right now. See my comment on moving the enum out of situational_play_selection? Or import in a different way?

Reply:
Yes. They started on individual lines, I just tested it again.

Reply:
```python
_situationList = [
    situational_play_selection.SituationalPlaySelector.Situation.ATTACK_GOAL,
    situational_play_selection.SituationalPlaySelector.Situation.DEFEND_CLEAR,
    situational_play_selection.SituationalPlaySelector.Situation.OFFENSIVE_SCRAMBLE
] # yapf: disable
```
Should do what you want.